examples/build_report.py

```diff
@@ -59,17 +60,21 @@
 squad_envs = Squad().environments(project=project.id)
 squad_suites = Squad().suites(project=project.id)
 envs = {}
+
 for env in squad_envs.values():
-    suites = {}
-    envs.update({env.slug: suites})
+    testjobs = {}
+    envs.update({env.slug: testjobs})
     tests = Squad().tests(test_run__environment=env.id, test_run__build=build.id)
-    # group tests by suite
-    for test in tests.values():
-        suite_name = squad_suites[test.suite].slug
-        if suite_name in suites.keys():
-            suites[suite_name].append(test)
-        else:
-            suites.update({suite_name: [test]})
+
+    for testrun, tests_set in groupby(sorted(tests.values(), key=lambda t: t.test_run), lambda t: t.test_run):
+        testjob = Squad().fetch_id(TestRun, id=testrun)['job_url']
```

Comment:
This got a bit weird `testjob = .... TestRun`, did you mean `testrun = ....TestRun`?
Also, `Squad().fetch_id()` feels like you're fetching the ID only. I had another idea about this that you want to accomplish, I'll leave a comment below.

Reply:
Well, in squad urls to tests use the job_id eg: https://qa-reports.linaro.org/lkft/linux-stable-rc-5.4-oe/build/v5.4.28/testrun/8388/suite/libhugetlbfs/tests/
and for Schneider case in particular they require job id to point to the actual testjob in lava.
Maybe we should add an arg to the report specifying whether to use hyperlinked urls for test job headers or not and if user chooses so, we use whatever squad uses?